src/main/java/mate/academy/rickandmorty/dto/internal/CharacterRickAndMortyDto.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.rickandmorty.dto.internal;
+
+import lombok.Data;
+
+@Data
+public class CharacterRickAndMortyDto {
```

Comment:
There is no need to include the application name into class naming
```suggestion
public class CharacterDto {
```

Reply:
+